ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml

```diff
@@ -47,14 +47,29 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: verify
-  commands: make test
+- as: fmt
+  commands: make verify-fmt
+  container:
+    from: src
+- as: vet
+  commands: make verify-vet
+  container:
+    from: src
+- as: lint
+  commands: make verify-lint
   container:
     from: src
 - as: vendor
   commands: make vendor
```

Comment:
Need to verify vendor too

Reply:
yes, I was confused by `go mod verify` command. it seems like it doesn't return  errors